lib/Jsrt/ChakraCommon.h

```diff
@@ -1505,7 +1797,7 @@
     /// <returns>
     ///     The code <c>JsNoError</c> if the operation succeeded, a failure code otherwise.
     /// </returns>
-    STDAPI_(JsErrorCode)
+    CHAKRA_API
         JsPointerToString(
```

Comment:
Why creating a new #ifdef _WIN32 block, should we move these 2 functions in the above #ifdef _WIN32 block?